.github/workflows/build-podman.yaml

```diff
@@ -132,6 +132,77 @@ jobs:
         docker-namespace: ${{ secrets.DOCKER_HUB_USER }}
         docker-password: ${{ secrets.DOCKER_HUB_SECRETS }}
         ghcr-token: ${{ secrets.GHCR_TOKEN }}
+        build-args: "NODE_VERSION=10.16.3"
+
+  BuildNodeJs14:
+    needs: BuildBase
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Docker meta for KubeSphere
+      id: meta
+      if: github.repository_owner == 'kubesphere'
+      uses: docker/metadata-action@v3
+      with:
+        images: |
+          kubespheredev/builder-nodejs
+          ghcr.io/${{ github.repository_owner }}/builder-nodejs
+        tags: |
+          type=schedule
+          type=ref,event=branch,suffix=-14-podman
+          type=ref,event=pr,suffix=-14-podman
+          type=semver,pattern=v{{version}}-14-podman
+    - name: Docker meta for Contributors
+      id: metaContributors
+      if: github.repository_owner != 'kubesphere'
+      uses: docker/metadata-action@v3
+      with:
+        images: |
+          ghcr.io/${{ github.repository_owner }}/builder-nodejs
+        tags: |
+          type=schedule
+          type=ref,event=branch,suffix=-14-podman
+          type=ref,event=pr,suffix=-14-podman
+          type=semver,pattern=v{{version}}-14-podman
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+    - name: Login to DockerHub
+      if: github.event_name != 'pull_request'
```

Comment:
Please refer to the custom GitHub Action to config the pipeline as follows:
https://github.com/kubesphere/devops-agent/blob/46d0e88a353dae4669386baf16d7241dd6a5039a/.github/workflows/build-podman.yaml#L119-L134